index.css

```diff
@@ -105,6 +105,15 @@ input[type="text"]::-ms-clear {
   outline: none;
 }
 
+.view-switcher {
+  display: none;
+} 
+
+@media (min-width: 768px){
+.view-switcher {
+  display: flex;
+}
```

Comment:
Curious why the linter didn't fix or catch this! Either way we should probably indent this.